redi-connect-backend/lib/email/templates/welcome-to-redi-munich.mjml

```diff
@@ -0,0 +1,26 @@
+<mjml>
+  <mj-include path="./head.mjml" />
```

Comment:
@mzdeloitte, nice, this kind of splitting into files makes a lot of sense.

Can you change the file names from `blablabla-munich.mjml` to `blablabla.munich.mjml`? That'll make the convention nice and uniform.